extensions/positron-r/src/test/debugger.test.ts

```diff
@@ -0,0 +1,89 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2025 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as assert from 'assert';
+import * as testKit from './kit';
+
+suite('Debugger', () => {
+	let sesDisposable: vscode.Disposable;
+
+	suiteSetup(async () => {
+		const [_ses, disposable] = await testKit.startR();
+		sesDisposable = disposable;
+	});
+
+	suiteTeardown(async () => {
+		if (sesDisposable) {
+			await sesDisposable.dispose();
+		}
+	});
+
+	test('Can debug in virtual namespace', async () => {
+		await testKit.withDisposables(async (disposables) => {
+			// Generate virtual namespace synchronously via `View()`
```

Comment:
```suggestion
			// Force virtual namespace generation via `View()`
			// This happens synchronously
```

I think _force_ is the keyword here about why `View()` is being used here.

Also worth considering whether ark could expose something like `.ps.generateVirtualNamespace(pkg)` that could be used here instead?